js/conversation_controller.js

```diff
@@ -205,9 +205,11 @@
       };
 
       conversation.initialPromise = create();
-      conversation.initialPromise.then(async () => {
-        await window.lokiSnodeAPI.refreshSwarmNodesForPubKey(id);
-        await conversation.updateProfileAvatar();
+      conversation.initialPromise.then(() => {
+        Promise.all([
+          conversation.updateProfileAvatar(),
+          window.lokiSnodeAPI.refreshSwarmNodesForPubKey(id),
```

Comment:
Just out of curiosity, does the order here matters?

Reply:
I think for our purposes no, but in reality what would happen is that the first call in the array would execute until there is an await and then the second call would start etc so they could interweave.
So if they were both referencing some shared state then it could get bad.
The important thing here is that currently the updateProfileAvatar call would be blocked until the refreshSwarmNodes call was complete which could take a long time

Reply:
So why not just keeping the code that `await`s each promises, but with the two calls swapped? I guess that would prevent having that "interweaving" execution right?

Reply:
Well with the promise all it would be ever so slightly more efficient because we could fire off the swarm request while the updateProfileAvatar call is accessing the database
Could just do them sequentially if u think would be easier to read?